.github/workflows/multi-gpu-ci.yml

```diff
@@ -0,0 +1,37 @@
+name: horovod (2GPU)
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  gpu-ci:
+    runs-on: 2GPU
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Install conda
+      run: |
+        mkdir -p /tmp/miniconda
```

Comment:
If we believe this is the only way, so be it. However, I would caution against this. Conda is not something we are trying to push people towards using. We are actually trying to move away from this. Are we sure this doesnt work in our environment. We had this testing in NVTabular before and those test ran https://github.com/NVIDIA-Merlin/NVTabular/blob/main/tests/unit/loader/test_tf_dataloader.py#L53. The cuda toolkit should exist in the users environment, that should be separate from installing merlin. I believe MPI should follow the same strategy. The user should have that installed. 
We really want to stay away from conda, if the user does not start by installing conda and then run all pip installs inside of that environment, the user wont have those packages. Also running in conda will be a terrible experience the moment you run into dependency problems. These are some of the reasons we are telling people to move away from that methodology. And On GPU CI running this will blow up the environment right (forcing the download of cuda toolkit which is around 1gig https://anaconda.org/anaconda/cudatoolkit/files)? We would essentially be creating this environment for every test run, right? I also see that the support for cuda toolkit stops at 11.3. Does this mean our environment will during testing will use 11.8 toolkit on the environment for cudf and 11.3 for horovod stuff? Given that we have the container with these requirements(cuda toolkit and MPI) already, do we need this? If our container does not have MPI and cuda toolkit, it might be better to add that to the container directly.